{
  "repository": {
    "name": "fabriciofx/cactoos-jdbc",
    "githubId": 132658324,
    "url": "https://github.com/fabriciofx/cactoos-jdbc",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "builds": {
    "buggyBuild": {
      "id": 416200068,
      "url": "http://travis-ci.org/fabriciofx/cactoos-jdbc/builds/416200068",
      "date": "Aug 15, 2018 5:56:53 AM"
    }
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Aug 15, 2018 5:57:25 AM",
    "reproductionDateEnd": "Aug 15, 2018 6:03:12 AM",
    "totalDuration": 344,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 3,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          3
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 155,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          0,
          155
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 24,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          24
        ]
      }
    },
    "projectRootPomPath": "./workspace/fabriciofx/cactoos-jdbc/416200068/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "tests": {
    "failingModule": "/root/workspace/fabriciofx/cactoos-jdbc/416200068",
    "overallMetrics": {
      "numberRunning": 19,
      "numberPassing": 11,
      "numberFailing": 1,
      "numberErroring": 7,
      "numberSkipping": 1,
      "failures": [
        {
          "failureName": "org.h2.jdbc.JdbcSQLException",
          "isError": true,
          "occurrences": 1
        },
        {
          "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
          "isError": true,
          "occurrences": 5
        },
        {
          "failureName": "java.lang.AssertionError",
          "isError": false,
          "occurrences": 1
        },
        {
          "failureName": "org.postgresql.util.PSQLException",
          "isError": true,
          "occurrences": 1
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "numberRunning": 3,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 3,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.TransactionTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.script.SqlScriptTest",
        "numberRunning": 2,
        "numberPassing": 1,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.server.PgSqlServerTest",
        "numberRunning": 1,
        "numberPassing": 0,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.UpdateTest",
        "numberRunning": 2,
        "numberPassing": 1,
        "numberFailing": 0,
        "numberErroring": 1,
        "numberSkipping": 0
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.query.SimpleQueryTest",
        "numberRunning": 5,
        "numberPassing": 4,
        "numberFailing": 1,
        "numberErroring": 0,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.UpdateTest",
        "testMethod": "longQuery",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.server.PgSqlServerTest",
        "testMethod": "start",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to localhost:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.stmt.TransactionTest",
        "testMethod": "agenda",
        "failureName": "org.h2.jdbc.JdbcSQLException",
        "detail": "Value too long for column \"CARRIER VARCHAR(10) NOT NULL\": \"\u002778c8605b-8767-4a94-b77b-f03cbf65afd9\u0027 (36)\"; SQL statement:\nINSERT INTO phone (contact, number, carrier) VALUES (?, ?, ?) [22001-197]",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.query.SimpleQueryTest",
        "testMethod": "outOfOrder",
        "failureName": "java.lang.AssertionError",
        "detail": "Expected exception: java.lang.IllegalArgumentException",
        "isError": false
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "testMethod": "addContact",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.script.SqlScriptTest",
        "testMethod": "mysql",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "testMethod": "findContact",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "com.github.fabriciofx.cactoos.jdbc.agenda.AgendaTest",
        "testMethod": "renameContact",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      }
    ]
  },
  "commits": {
    "buggyBuild": {
      "repoName": "fabriciofx/cactoos-jdbc",
      "branchName": "mysql",
      "sha": "8c03ab02ab1e6579c4346e430f6cc47bb49726f6",
      "url": "http://github.com/fabriciofx/cactoos-jdbc/commit/8c03ab02ab1e6579c4346e430f6cc47bb49726f6",
      "date": "Aug 15, 2018 5:24:48 AM"
    }
  },
  "projectMetrics": {
    "numberModules": 1,
    "numberSourceFiles": 69,
    "numberTestFiles": 34,
    "numberLibrariesFailingModule": 28
  },
  "type": "only_fail"
}